### PR TITLE
Kalyani into master

### PR DESCRIPTION
**Pull request by abhinaykrupa**

### CODE REVIEW

File: application/models/BoatsModel.php [hunk 1]
```diff
@@ -196,9 +207,8 @@ class boatsModel extends CI_Model
             'all' => '',
         ],
         'boatexportusa' => [
-            'new' => '',
-            'used' => 'used_boattrader/listings',
-            'certified' => 'used_boattrader/listings',
+            'search_in' => 'tittle',
+            'keyword'   => 'keyword=make',
```
Comment (abhinaykrupa) [inline application/models/BoatsModel.php]: @kadarikalyani  keyword=make ani malli declare cheyyalisina avasaram ledu anukunta .... just "keyword => make" ante chalu kada ... take a look

File: application/models/BoatsModel.php [hunk 2]
```diff
@@ -340,7 +332,7 @@ public function getWebsites($params)
                 }
                 if ($params['priceMin'] || $params['priceMax']) {
                     $this->query[$website_id]['pricerange'] = ($params['priceMin'] ? $params['priceMin'] : '0').'-'
-                        .($params['priceMax'] ? $params['priceMax'] : 'max');
+                        .($params['p]riceMax'] ? $params['priceMax'] : 'max');
```
Comment (abhinaykrupa) [inline application/models/BoatsModel.php]: @kadarikalyani  typo here .. take a look

File: application/models/BoatsModel.php (hunk 1 above)
Comment (kadarikalyani) [inline application/models/BoatsModel.php]: yeah ok

File: application/models/BoatsModel.php (hunk 2 above)
Comment (kadarikalyani) [inline application/models/BoatsModel.php]: ok

**Pull request merged**